networkit/test/test_component.py

```diff
@@ -25,6 +25,36 @@ def testNode(v):
 				cc.run()
 				self.assertEqual(cc.numberOfComponents(), 1)
 			G1.forNodes(testNode)
+
+	def testBiconnectedComponentsSize(self):
+		G = nk.Graph(5)
+		for i in range(2):
+			G.addEdge(i, i+1)
```

Comment:
I suggest to keep things simple :) same for the others instances below.
```suggestion
		G.addEdge(0, 1)
		G.addEdge(1, 2)
```